willow/plugins/pillow.py

```diff
@@ -425,7 +425,12 @@ def save_as_heic(
     def save_as_avif(self, f, quality=80, lossless=False, apply_optimizers=True):
         kwargs = {"quality": quality}
         if lossless:
-            kwargs = {"quality": -1, "chroma": 444}
+            kwargs = {
+                # Quality of 100 implies lossless (according to libavif documentation)
+                "quality": 100,
+                # When encoding lossless images, don't use chroma subsampling (4:4:4 retains the most information)
+                "subsampling": "4:4:4",
```

Comment:
This won't actually be valid when encoding a grayscale image, should be `4:0:0` instead (grayscale is Pillow `L` or `LA` mode image) - something for a follow up?

Reply:
let's do it as a follow up (as long as you have the time :D)

Reply:
Tried writing a failing test but it turns out `save_as_avif` will convert from `LA` to `RGBA` internally - which makes this not really relevant... or at least not without same yak shaving to figure out why the conversion is happening.

Let's leave this here for now.